compose.yaml

```diff
@@ -319,5 +319,13 @@ services:
       airflow-init:
         condition: service_completed_successfully
 
```

Comment:
In development we can serve up the static site using nginx. If we had another webserver that was running that could serve the website out of the Rialto volume that would work for our QA, Stage and Production environments?